### DISCUSSION TITLE
Changes to campaigns require reloading

### DISCUSSION THREAD

**Discussion by jimkohl** [LefterisXris/CodeTour#87] [General]
After changing titles, etc. or deleting, need to restart IntelliJ - now that I know it I know but would be better if it updated on chg.

**Comment by LefterisXris** [comment]
Hey @jimkohl, restart is not required. Whenever you do changes from Tours pane, reload is automatically triggered. 
However, if you make changes directly to `tour` files, you should use the `Reload` button at the bottom of Tours Pane

![image](https://github.com/LefterisXris/CodeTour/assets/22836249/e26c752b-33dd-4f85-8b1f-3c802875a99f)

You can actually click **Reload** whenever you want to actually reload tours. Just note that there is an issues with Panel focus, so you may need to click on the editor (so that Tours Panel lose focus) and re-render the tours properly.
Also, I am going to provide the `Reload` option in the wider context menu soon: https://github.com/LefterisXris/CodeTour/issues/83
